terraform/main.tf

```diff
@@ -84,7 +84,10 @@ resource "null_resource" "apply_deployment" {
 resource "null_resource" "wait_conditions" {
   provisioner "local-exec" {
     interpreter = ["bash", "-exc"]
-    command     = "kubectl wait --for=condition=ready pods --all -n ${var.namespace} --timeout=-1s"
+    command     = <<-EOT
+    kubectl wait --for=condition=AVAILABLE apiservice/v1beta1.metrics.k8s.io --timeout=1200s
+    kubectl wait --for=condition=ready pods --all -n ${var.namespace} --timeout=1200s
+    EOT
   }
```

Comment:
The use of [`<<-EOT` is documented here](https://developer.hashicorp.com/terraform/language/expressions/strings#indented-heredocs).